src/plugins/login/loginPlugin.test.ts

```diff
@@ -19,7 +19,7 @@ describe("Login Plugin", () => {
     const options = MockFactory.createTestServerlessOptions();
```

Comment:
this login order change has nothing to do with webpack, I would put this in a different pr

Reply:
Reverted login changes.